### PR TITLE
Make EscapeSyntax respect encodeHtml

### PR DESCRIPTION
**Pull request by TarekkMA**
fixes: #385

### CODE REVIEW

**Comment by kevmoo** [comment]
Looks like you need to format the code @TarekkMA !

**Comment by TarekkMA** [comment]
Fixed.

File: test/document_test.dart [hunk 1]
```diff
@@ -82,6 +96,20 @@ void main() {
         expect(
             codeBlock.textContent, equals('<p>Hello <em>Markdown</em></p>\n'));
       });
+
+      test('leave double quotes, grater than, and less than when escaped', () {
```
Comment (kevmoo) [inline test/document_test.dart:100]:
```suggestion
      test('leave double quotes, greater than, and less than when escaped', () {
```

**Review by srawlins — APPROVED** [review]
Thanks! Approved with kevmoo's last request

**Review by srawlins — APPROVED** [review]

**Comment by TarekkMA** [comment]
Seems like the typo induced a formatting change.

**Comment by kevmoo** [comment]
@srawlins – we should bump the version here and have a changelog entry, no?

**Pull request merged**